pycqed/analysis/measurement_analysis.py

```diff
@@ -3713,9 +3713,9 @@ def run_default_analysis(self, print_fit_results=False,
             amplitude_guess = amplitude_factor * np.pi*kappa_guess * abs(
                 max(self.measured_powers)-min(self.measured_powers))
 
-            LorentzianModel.set_param_hint('f0', value=f0,
-                                           min=min(self.sweep_points),
-                                           max=max(self.sweep_points))
+            LorentzianModel.set_param_hint('f0', value=f0*1e-9,
```

Comment:
In https://github.com/DiCarloLab-Delft/PycQED_py3/pull/118/commits/2e4bfa4ff51b87b795dc5a600b28beaac94a6400 I changed units to SI in all code i wrote/changed. This includes the LorentzianModel which is used by Qubit_Spectroscopy_Analysis. I made the obvious corrections there, but I am not sure it didn't break anything anywhere else.